components/style/gecko_bindings/check_bindings.py

```diff
@@ -0,0 +1,32 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
```

Comment:
Uh, sorry for the confusion again.

To be clear: `check_bindings.py` belongs in `binding_tools`. The generated file belongs in `gecko_bindings`.